pkg/alerts/alertsHandler/cronJobHandler.go

```diff
@@ -218,8 +221,31 @@ func handleAlertCondition(alertToEvaluate *alertutils.AlertDetails, isAlertCondi
 	return nil
 }
 
+func getLogsQueryLinkForTheAlert(alertDetails *alertutils.AlertDetails, timeRange *dtypeutils.TimeRange) string {
+	httpScheme := "http://"
+
+	if config.IsTlsEnabled() {
+		httpScheme = "https://"
+	}
+
+	baseURL := httpScheme + config.GetQueryHostname() + ":5122/index.html"
```

Comment:
My assumption of `:5122` might not be true for all the scenario's. Users can host SigLens through a proxy server and their query link can be just: `https://example.com/index.html?searchText=*&....`

How do we want to resolve this?

Reply:
Or do you think, the host name will be given as `example.com:5122`? If SigLens is being server through that port?